spec/system/page_objects/pages/user.rb

```diff
@@ -24,6 +24,22 @@ def has_warning_messages_path?(user)
         page.has_current_path?("/u/#{user.username}/messages/warnings")
       end
 
+      def has_primary_navigation_item?(name)
+        page.has_css?(".user-navigation-primary li.user-nav__#{name}")
```

Comment:
Only thing I would suggest is having this

```ruby
def primary_navigation_selector(name)
  ".user-navigation-primary li.user-nav__#{name}"
end
```

Then if it changes you only have to do one place.

Reply:
Thanks! I've changed it